iroh-p2p/src/config.rs

```diff
@@ -42,8 +42,10 @@ pub struct Libp2pConfig {
     pub bootstrap_peers: Vec<Multiaddr>,
     /// Mdns discovery enabled.
     pub mdns: bool,
-    /// Bitswap discovery enabled.
-    pub bitswap: bool,
+    /// Bitswap server mode enabled.
+    pub bitswap_server: bool,
+    /// Bitswap client mode enabled.
+    pub bitswap_client: bool,
```

Comment:
I'll annotate that these are default true on the docs site. I'd ask for a `default: true` in the comment, but there's too high a risk it'll de-syncronize from the `default` method over time